compiler/parser.cpp

```diff
@@ -5670,10 +5660,10 @@ compound(int stmt_sameline) {
 
     testsymbols(&loctab, nestlevel, FALSE, TRUE); /* look for unused block locals */
     declared = save_decl;
-    delete_symbols(&loctab, nestlevel, FALSE, TRUE); /* erase local symbols, but
-                                                      * retain block local labels
-                                                      * (within the function) */
-    nestlevel -= 1;                                  /* decrease compound statement level */
+    delete_symbols(&loctab, nestlevel, TRUE); /* erase local symbols, but
+                                               * retain block local labels
+                                               * (within the function) */

```

Comment:
This comment is now useless.  It is extremely important that we remove it.

Reply:
Done!